unpinned_requirements.txt

```diff
@@ -4,8 +4,8 @@ pandas
 scikit-learn
 torch
 spacy<2.3.0,>=2.2.1
-#spacy-transformers
--e git+https://github.com/nsorros/spacy-transformers#egg=spacy-transformers
+spacy-transformers
+#-e git+https://github.com/nsorros/spacy-transformers#egg=spacy-transformers
```

Comment:
do you still need it in the unpinned requirements?

Reply:
i thought i removed that

Reply:
it is commented out but will remove